src/libraries/System.Text.RegularExpressions/tests/RegexSRMTests.cs

```diff
@@ -1001,5 +1001,14 @@ public static IEnumerable<object[]> ValidateSRMRegex_NotSupportedCases_Data()
             yield return new object[] { @"\G(\w+\s?\w*),?", RegexOptions.None, "contiguous matches" };
             yield return new object[] { @"(?>a*).", RegexOptions.None, "atomic" };
         }
+
+        [Fact]
+        public void StressTestSymbolicRegexConversion()
+        {
+            string pattern = string.Concat(Enumerable.Repeat("([a-z]", 1000).Concat(Enumerable.Repeat(")", 1000)));
```

Comment:
does this fail on NFA? If not shall we make it generic to both

Reply:
Variant of this one already appears as part of Replace tests (in Replace_String_TestData). So it is covered in backtracking case as well. We had an SRM meeting today with Olli and discussed that over time we should collect and add more such stress tests, say with deeply nested loops etc -- as robustness tests on the initial parsing/conversion phase, also for the BDD algebra.

Reply:
I added None and Compiled in any case because the test here is slightly different.